bundle/manifests/falcon-helm.clusterserviceversion.yaml

```diff
@@ -0,0 +1,258 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "falcon.crowdstrike.com/v1alpha1",
+          "kind": "FalconSensor",
+          "metadata": {
+            "name": "falcon-helm"
+          },
+          "spec": {
+            "falcon": {
+              "aid": null,
+              "apd": null,
+              "aph": null,
+              "app": null,
+              "assert": null,
+              "billing": null,
+              "cid": null,
+              "feature": null,
+              "memfail_every_n": null,
+              "memfail_grace_period": null,
+              "message_log": null,
+              "provisioning_token": null,
+              "tags": null,
+              "trace": null,
+              "update": null
```

Comment:
Aren't these set somewhere else? Seems like we're defining these variables in multiple places?

Reply:
The operator-sdk generates and manages the `bundle` directory and all files under it based on what is in the `config` directory. So, yes, the file is elsewhere, but this one is generated into the manifest based on what is under `config`.